tests/test_settings.py

```diff
@@ -511,12 +512,33 @@ def test_postgres_database_settings_may_be_set_individually(self):
                 PREFECT_API_DATABASE_PASSWORD: "the-password",
             }
         ):
-            assert PREFECT_API_DATABASE_CONNECTION_URL.value() == (
-                "postgresql+asyncpg://"
-                "the-user:the-password@"
-                "the-database-server.example.com:15432"
-                "/the-database"
-            )
+            url = make_url(PREFECT_API_DATABASE_CONNECTION_URL.value())
+            assert url.drivername == "postgresql+asyncpg"
+            assert url.host == "the-database-server.example.com"
+            assert url.port == 15432
+            assert url.username == "the-user"
+            assert url.database == "the-database"
+            assert url.password == "the-password"
```

Comment:
I decided to change the tests to parse the URL instead of just string match it, since that would uncover quoting issues.